packages/network/package.json

```diff
@@ -75,7 +75,7 @@
   },
   "scripts": {
     "build": "coffee -b -o lib src && sed -i -e 's/src/lib/g' lib/register.js",
-    "pretest": "npm build",
+    "pretest": "npm run build",
```

Comment:
Use `yarn`

Reply:
If we use yarn, we have to install it in every Docker environment, right?

Reply:
This issue is about testall, using yarn is docker is another topic

Reply:
done